### DISCUSSION TITLE
Introducing GitHub Enterprise Importer

### DISCUSSION THREAD

**Discussion by tuves** [community/community#55519] [Enterprise]
Today, we're launching a new brand new tool for migrating from other code hosting platforms to GitHub and between GitHub products: GitHub Enterprise Importer (GEI).

With GitHub Enterprise Importer, you can migrate to GitHub.com or GitHub Enterprise Cloud and bring your source code and collaboration history (for example code reviews and comments) with you.

We’re publicly launching GitHub Enterprise Importer today — but already, it has been used by over 2,000 customers to migrate more than 400,000 repositories to GitHub Enterprise Cloud.

Read more in our [changelog](https://github.blog/changelog/2023-06-12-introducing-github-enterprise-importer/).

**Comment by faceboock1** [comment]
Thanks

**Comment by MauraDK** [comment]
Sounds like a great 👍 I d ah! Thanks 😊

**Comment by Alex-ley-scrub** [comment]
We can't get this to work even with Full Access PATs to migrate from AzDO to GH
